ui/src/main/webapp/app.js

```diff
@@ -16,6 +18,62 @@ const theme = createMuiTheme({
   },
 })
 
+const Banner = props => {
+  const { children } = props
+
+  return (
+    <Query query={getUiConfig}>
+      {({ data, loading, error }) => {
+        if (loading) return <p>Loading...</p>
+        if (error) return <p>Error...</p>
```

Comment:
❗️ We should either continue rendering the rest of the app here, ie ignore the error.